content/guides/core/automations/automation-events.md

```diff
@@ -41,11 +41,43 @@ You can create a project automation at these scopes:
 - The project. The automation watches for the event taking place on any collection in the project.
 - All collections in the project that match the filter you supply.
 
-### Events
-A project automation can watch for these events:
+### Artifact events
+An automation can watch for these [artifact]({{< relref "/guides/core/artifacts/" >}}) events:
 - **A new version of an artifact is created in a collection**: Apply recurring actions to each version of an artifact. Specifying a collection is optional. For example, start a training job when a new dataset artifact version is created.
 - **An artifact alias is added**: Trigger a special step of your workflow when a new artifact version in a project or collection has a specific alias applied. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied.
 
+### Run metrics events
+An automation can watch for changes in a run metric:
+
+- **Run metrics threshold met**: Trigger a workflow when for a given metric, a single logged value or the average logged values meets the threshold you specify.
+- **Run metrics change threshold met**: Trigger a workflow when for a given metric, a single logged value or the average logged value changes by the threshold you specify.
+
+{{% alert %}}
+You can create aautomations triggered by system metrics such as the `cpu` metric, which tracks the percentage of CPU utilization. W&B logs system metrics automatically every 15 seconds. Find each metric's name and details in [System metrics]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}).
+{{% /alert %}}
+
+For a run metric event, you can configure how to compare the run metric value with the threshold you specify. Your choices depend on the event type.
+
+#### Threshold
+For **Run metrics threshold met** events, you configure:
+1. The number of logged values to average across (defaults to 5).
+1. How to compare the values with the threshold.
+
+For example, trigger an automation when `accuracy` exceeds `.6`.
+
+#### Change threshold
+
+For **Run metrics change threshold met** events, you can configure:
+1. The current window of logged values to average across (defaults to the 10 most recent logged values).
+1. The prior window of logged values to average across (defaults to the 50 most recent logged values prior to the current window).
+1. Whether to evaluate the values as relative or absolute (defaults to **Relative**).
+1. How to compare the values with the threshold:
+      - Increases by at least
+      - Decreases by at least
+      - Increases or decreases by at least
+
+The current and prior window do not overlap. By default, the current window includes the 10 most recent logged values, and the prior window includes the 50 logged values prior to the current window.
```

Comment:
also @mdlinville I realized something I'm working on could affect these docs - not sure if you wanted to change them at all 

https://app.graphite.dev/github/pr/wandb/core/28397/feat(artifacts)-Simplified-alert-creation-UI-for-workspaces is the PR

basically, just in workspaces we have a simplified UI for creating slack notifications when you're making automations on a metric.

Reply:
I'm having trouble accessing that. If it is not already available, we should address it as a fast follow. These docs are intended to be published very very soon.